src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/AsyncLazy`1.cs

```diff
@@ -13,6 +13,12 @@
 
 namespace Roslyn.Utilities
 {
+    internal static class AsyncLazy
```

Comment:
❔ Why?

Reply:
makes construction much nicer.  you can just do `AsyncLazy.Create(c => ComputeAsync(...))` isntead of `new AsyncLazy<ImmutableDictionary<SchemeName, ImmutableArray<RegistryItem>>>` (an actual example in our code).